app/controllers/boards_controller.rb

```diff
@@ -0,0 +1,6 @@
+class BoardsController < ApplicationController
+
+	def show
+	end
+
+end
```

Comment:
Ruby files are generally indented using two spaces rather than a tab. I realize this sounds nitpicky, but it's a very common convention, so you may catch some flak for not following it. You can set up Sublime to enforce this for you by opening up a Ruby file, going to your syntax specific settings, and adding: 
```
"tab_size": 2,
"translate_tabs_to_spaces": true 
```
With these settings in place, whenever you use a tab in an `.rb` file, it'll automatically get converted into two spaces.